src/main/java/labs/introtoprogramming/lab5/object/Builder/POJO/Material.java

```diff
@@ -0,0 +1,200 @@
+package labs.introtoprogramming.lab5.object.Builder.POJO;
+
+import labs.introtoprogramming.lab5.object.Builder.BuilderInterface;
+
+public class Material {
```

Comment:
This class has too many information in it. Simplify it. Also remove number of setters and getters. It is bad practice to put them. Move as much dependencies as possible to constructors